policyengine_us/variables/gov/states/or/ohcs/liheap/or_liheap_benefit_level.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class or_liheap_benefit_level(Variable):
+    value_type = int
+    entity = SPMUnit
+    label = "Income range for Oregon LIHEAP eligibility"
+    definition_period = YEAR
+    reference = "https://www.oregon.gov/ohcs/energy-weatherization/Documents/2021-Energy-Assistance-Manual.pdf#page=55"
+
+    defined_for = StateCode.OR
+
+    def formula(spm_unit, period, parameters):
+        income = add(spm_unit, period, ["irs_gross_income"])
+        threshold = spm_unit("or_liheap_income_threshold", period)
+        p = parameters(period).gov.states["or"].ohcs.liheap
+
+        level_one = threshold * p.or_liheap_benefit_level_multiplier
+        level_two = threshold * p.or_liheap_benefit_level_multiplier * 2
+        level_three = threshold * p.or_liheap_benefit_level_multiplier * 3
```

Comment:
Are the 2 and 3 policy parameters? i am tempted to ask to parameterize here 

And we should probably convert this into an ENUM file - this will make the parameter file cleaner as well

Reply:
After speaking with Ziming -> if we mathematically derived, can we leave a comment and link to the spreadsheet

Reply:
similar question persists